mantidproject/mantid
# patch for HB2A concerning data save in GSAS format

**Description of work.**

The autoreduction script for HB2A is using `SaveGSSCW` to save GSAS formatted data whereas the `HB2AReduce` script was using `SaveGSS` to save GSAS formatted data. Current pull request is to change `HB2AReduce` algorithm to be consistent with autoreduction workflow in terms of saving GSAS formatted data.



**To test:**

Launch `HB2AReduce` algorithm and put in an example run as shown below,

![hsvnmHAg](https://user-images.githubusercontent.com/2872152/110495184-43891480-80c2-11eb-9616-36203d05c1fe.png)

Check the output to see whether it is following the same format as data saved from autoreduction, as given below,

[autoreduction.zip](https://github.com/mantidproject/mantid/files/6109594/autoreduction.zip)





*There is no associated issue.*





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
